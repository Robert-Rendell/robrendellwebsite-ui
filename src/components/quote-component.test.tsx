import React from "react";
import { render } from "@testing-library/react";
import { QuoteComponent } from "./quote.component";

describe("QuoteComponent", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<QuoteComponent year={2023}/>);
    expect(asFragment()).toMatchSnapshot();
  });
});