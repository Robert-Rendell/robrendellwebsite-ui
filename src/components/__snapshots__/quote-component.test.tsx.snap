// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuoteComponent should match snapshot 1`] = `
<DocumentFragment>
  <h2
    class="centred"
  >
    <i>
      ""
    </i>
  </h2>
  <p
    class="centred"
  >
    <i>
      Rob Rendell (2023)
    </i>
  </p>
</DocumentFragment>
`;
